01-path-basics/102-your-first-cluster/readme.adoc

```diff
@@ -68,6 +68,54 @@ echo $EKS_SECURITY_GROUPS
 ```
 If any of those environment variables are blank, please re-run the "Build Script" section of the link:../101-start-here[Cloud9 Environment Setup].
 
+If you receive an *UnsupportedAvailabilityZoneException* error during EKS cluster creation, your account is using an AZ that is currently resource constrained. This occurs mostly in N.Virginia region (us-east-1).
+
+```
+An error occurred (UnsupportedAvailabilityZoneException) when calling the CreateCluster operation: Cannot create cluster 'k8s-workshop' because us-east-1c,
+the targeted availability zone, does not currently have sufficient capacity to support the clust er. Retry and choose from these availability zones: us-east-1a, us-east-1b, us-east-1d
+``` 
+
+If you receive this error, you need to remove the constrained AZ (us-east-1c in my case) from *`EKS_SUBNET_IDS`* environment variable. Follow these steps to update your environment variable
+
+Save the EKS recommended AZ's that is referred in your CLI output in an environment variable.
+Note: you only need two AZ's defined to create EKS cluster
+
+    $ export EKS_VALID_AZS=us-east-1a,us-east-1b
+
+Run the command below to determine subnet ID's
+
+    $ aws ec2 describe-subnets --filters "Name=vpc-id,Values=$EKS_VPC_ID" "Name=availabilityZone,Values=$EKS_VALID_AZS" --query 'Subnets[*].[SubnetId]' --output text
+    subnet-6e672524
+    subnet-18b10e44
+
+Save this output as `*EKS_SUBNET_IDS*` environment variable
+
+    $ export EKS_SUBNET_IDS=subnet-6e672524,subnet-18b10e44
+
+Re-run EKS create-cluster and you should now be able to create cluster. The output should look similar to this
+
+    {
+    "cluster": {
```

Comment:
Indent lines 98-116 for proper Pretty JSON.

Reply:
all issues fixed in latest commit. The text seems indented in my commit. Using Atom and adoc formatting for indentation. Can you check and lmk if you still have indentation issue?